.proxy-version

```diff
@@ -1 +1 @@
-v2.199.0
+v2.200.0
```

Comment:
let's update this to pick up the bump to v2.201.0 that just merged? https://github.com/linkerd/linkerd2/commit/51541d0ddb4870f8298b75f950200bbd586f08e1

Reply:
can we save that for the edge release and not the stable release?

Reply:
we can! i thought that we had previously discussed wanting to merge this change for 2.13.3, but it's fine if we don't want to do that.